common/flavorization/wc_title_holders.txt

```diff
@@ -162,149 +152,2026 @@ warlord_female = {
+	special = councillor
+	tier = duchy
+	priority = 299
+	council_position = councillor_chancellor
+	cultures = { mantid }
+}
+vizier_female_mantid = {
+	type = character
+	gender = female
+	special = councillor
+	tier = duchy
+	priority = 299
+	council_position = councillor_chancellor
+	cultures = { mantid }
+}
+
+### Qiraji ###
+# Special
+emperor_ahn_qiraj_male_qiraji {
+	type = character
+	gender = male
+	special = holder
+	tier = kingdom
+	priority = @kingdom_cultural_prio
+	top_liege = no
+	cultures = { qiraji }
+	titles = { k_ahnqiraj }
+}
+empress_ahn_qiraj_female_qiraji {
+	type = character
+	gender = female
+	special = holder
+	tier = kingdom
+	priority = @kingdom_cultural_prio
+	top_liege = no
+	cultures = { qiraji }
+	titles = { k_ahnqiraj }
+}
+empire_ahn_qiraj = {
+	type = title
+	tier = kingdom
+	priority = @kingdom_cultural_prio
+    cultures = { qiraji }
+	titles = { k_ahnqiraj }
+}
+# Others
+lord_male_qiraji_count = {
+    type = character
+	gender = male
+	special = holder
+	tier = county
+	priority = @county_cultural_prio
+	top_liege = no
+	cultures = { qiraji }
+}
+lady_female_qiraji_count = {
+    type = character
+	gender = female
+	special = holder
+	tier = county
+	priority = @county_cultural_prio
+	top_liege = no
+	cultures = { qiraji }
+}
+hive_qiraji_county = {
+	type = title
+	tier = county
+	priority = @county_cultural_prio
+	cultures = { qiraji }
+}
+high_overlord_male_qiraji_duke = {
+    type = character
+	gender = male
+	special = holder
+	tier = duchy
+	priority = @duchy_cultural_prio
+	top_liege = no
+	cultures = { qiraji }
+}
+high_overlady_female_qiraji_duke = {
+    type = character
+	gender = female
+	special = holder
+	tier = duchy
+	priority = @duchy_cultural_prio
+	top_liege = no
+	cultures = { qiraji }
+}
+brood_qiraji_duchy = {
+	type = title
+	tier = duchy
+	priority = @duchy_cultural_prio
+	cultures = { qiraji }
+}
+
+### Wildhammer ###
+# Feudal
+chief_male_wildhammer_count = {
+    type = character
+	gender = male
+	special = holder
+	tier = county
+	priority = @county_cultural_prio
+	top_liege = no
+	governments = { feudal_government tribal_government clan_government }
+	cultures = { wildhammer }
+	faiths = { shamanism }
+}
+chieftess_female_wildhammer_count = {
+    type = character
+	gender = female
+	special = holder
+	tier = county
+	priority = @county_cultural_prio
+	top_liege = no
+	governments = { feudal_government tribal_government clan_government }
+	cultures = { wildhammer }
+	faiths = { shamanism }
+}
+chiefdom_wildhammer_county = {
+	type = title
+	tier = county
+	priority = @county_cultural_prio
+	governments = { feudal_government tribal_government clan_government }
+	cultures = { wildhammer }
+	faiths = { shamanism }
+}
+thane_male_wildhammer_duke = {
+    type = character
+	gender = male
+	special = holder
+	tier = duchy
+	priority = @duchy_cultural_prio
+	top_liege = no
+	governments = { feudal_government tribal_government clan_government }
+	cultures = { wildhammer }
+}
```

Comment:
Why you haven't created one Thane entity and assigned all cultures that use thane: vrykul, kvaldir, etc. The same questions about other titles like High King.

Reply:
On CK3 coop I was advised not to do it because it is apparently less efficient with what Paradox code has.
Without forgetting that it is less clear to navigate.

Reply:
Fine.

Reply:
`high_king_male_old_dwarven_emperor` and `high_king_male_emperor` do the same thing twice.

Reply:
done in ac0afe713